### DISCUSSION TITLE
In disaggregateDomain() - SMCRT depleted when running the main version wrf-hydro with crocus

### DISCUSSION THREAD

**Discussion by Lise-Li** [NCAR/wrf_hydro_nwm_public#745] [Q&A]
Hi, wrf-hydro community.:
I'm running the WRF-Hydro model with crocus on. With the observation data from 2009-2010, I changed some parameters (RETDEPRTFAC=0.001/REFKDT=0.1/MAXSMC=0.8/SLOPE=1/DKSAT=10/rsurfexp=1/vcmx25=0.6/mp=0.6/CMPVT=1.8/MFSNO=3.3/Zmax=25/Expon=3/OVROUGHRTFAC=0.1/MannN=2), and the model can run successfully. But I run the model from 2004-2018 with the same parameters I tuned based on the observation, I got the error" In disaggregateDomain() - SMCRT depleted". It seemed that something wrong in  the Noah_distr_routing.F file, but I don't know how to solve it. Would you mind help me? Looking forward to your reply!
Best wishes,
Lise Li

**Comment by aubreyd** [comment]
Hi Lise Li: It is hard for me to diagnose the exact error, but I will note that some of your parameters are outside of what I would consider standard ranges. DKSAT = 10 indicates a saturated conductivity of 10 m/s - this is FAR bigger than the standard estimates (e.g., 5.23E-6 m/s for a sandy loam). MAXSMC = 0.8 means your soil column can reach 80% volumetric water content (if you lookup values for porosity, these generally max out at maybe a bit over 0.5). RETDEPRTFAC = 0.001 is smaller than I generally use, but I doubt that would cause a crash. I would recommend you bring your DKSAT back within range, and probably SMCMAX as well.

**Reply by Lise-Li** [reply]
Hi, @aubreyd 
Thank you for your reply! I tuned these parameters based on some references which may have something wrong, and I would like to know the standard ranges in your opinion. Since DKSAT/MAXSMC are series values, the DKSAT and MAXSMC I mentioned above refer to multiplers of the default values. I would appreciate it if you could tell me the specific acceptable range of each parameter. Looing forward to your reply!
Best wishes,
Lise Li

**Reply by aubreyd** [reply]
Hi Lise Li:
You can check this slide deck for some guidance on parameters: https://ral.ucar.edu/sites/default/files/public/projects/wrf-hydro/training-materials/Overview_of_Model_Calibration_Arezoo.pdf
Which version of the WRF-Hydro code are you running?

**Reply by Lise-Li** [reply]
Hi, @aubreyd 
Thank you for providing the slide deck for me, and I checked that my parameters were all within ranges. The WRF-Hydro code I used was the main version. I tried to upload my model documention to the attachment, but it's too big. Would you mind telling me your email address if iti is convenient? I'll send my model documention to you. I would appreciate it if you could helping  me seeing what's wrong with my model.
Best wishes,
Lise Li

**Reply by aubreyd** [reply]
You can email our help alias at wrfhydro@ucar.edu. I just noticed that you are using Crocus, which is still somewhat under development so not fully tested/supported. Please try running your domain with Crocus turned off and see if you get the same error.

**Reply by Lise-Li** [reply]
OK，thank you for your kindly help!

**Reply by Lise-Li** [reply]
Hi, @aubreyd 
I tried running my domain with Crocus turned off, and the model was successful. So the problem may be related with the Crocus. Thank you for your advice. 
Best wishes,
Lise Li

**Reply by aubreyd** [reply]
Thanks for the update. Tagging @trudeeidhammer, FYI.

**Reply by trudeeidhammer** [reply]
I believe the error might come from the IVGTYP value in the wrfinput file when running with Crocus. I saw the same thing with one of my runs. In every gridcell for where glacier =1, IVGTYP should be set to 23 (bare ground). I think that should fix the problem
